activestorage/package.json

```diff
@@ -16,18 +17,21 @@
   },
   "author": "Javan Makhmali <javan@javan.us>",
   "license": "MIT",
+  "dependencies": {
+    "spark-md5": "^3.0.0"
+  },
   "devDependencies": {
     "babel-core": "^6.25.0",
     "babel-loader": "^7.1.1",
     "babel-preset-env": "^1.6.0",
     "eslint": "^4.3.0",
     "eslint-plugin-import": "^2.7.0",
-    "spark-md5": "^3.0.0",
     "webpack": "^3.4.0"
   },
   "scripts": {
     "prebuild": "yarn lint",
     "build": "webpack -p",
-    "lint": "eslint app/javascript"
+    "lint": "eslint app/javascript",
+    "prepublishOnly": "mkdir -p src && cp app/javascript/activestorage/*.js src/"
```

Comment:
Let's make sure `src/` doesn't have any stale files in it: `rm -rf src && cp -R app/javascript/activestorage src`

Reply:
Good idea. Addressed in 0e89643a7038c8b4d4e1aad0efa8ef759473aa06